lib/osf-components/addon/components/contributors/widget/template.hbs

```diff
@@ -8,6 +8,9 @@
         <label for={{currentContributorFieldId}}>
             {{t 'osf-components.contributors.currentContributors'}}
         </label>
+        <div local-class='warning-container'>
```

Comment:
Do we need to add logic for this to only show up on the create workflow and not on the edit workflow?